src/utils/models.py

```diff
@@ -2,6 +2,12 @@
 import torch.nn.functional as F
 
 class ConvNet_1(nn.Module):
+    """
+    A simple Convolutional Neural Network with two convolutional layers,
+    followed by three fully connected layers for classification. This model 
+    uses ReLU activations and max pooling after each convolution.
```

Comment:
Please create separate files for each neural net, it will be more readable and structured.